common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -224,6 +224,12 @@ function(HTML5Video, HTML5HLSVideo, Resizer, HLS, _, Time) {
         if (state.isTouch) {
             dfd.resolve();
         }
+
+        if (document.getElementById("unit-next-button").disabled === false)
```

Comment:
Use jquery and class selector
`$('.sequence-nav-button.button-next').prop('disabled')`